pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -214,3 +234,138 @@ func (s *completed) Execute(msg dispatcher.DIDCommMsg) (state, error) {
 		return nil, fmt.Errorf("illegal msg type %s for state %s", msg.Type, s.Name())
 	}
 }
+
+func sendExchangeRequest(msg dispatcher.DIDCommMsg, outbound dispatcher.Outbound, doc *did.Doc) error {
+	requestBytes, dest, err := createExchangeRequest(msg.Payload, doc)
+	if err != nil {
+		return err
+	}
+	//send the exchange request
+	err = outbound.Send(requestBytes, dest)
```

Comment:
outbound.Send() was changed to accept `interface{}` instead of `[]byte` - see [here](https://github.com/hyperledger/aries-framework-go/blob/cc77a924dfd31555699a6992753a4a7936196d12/pkg/didcomm/dispatcher/api.go#L37).

Reply:
refactored